Add section about log/error messages to HACKING.md (#276)
canonical/pebble: HACKING.md

)
```

### Log and error messages

> **Log messages** (i.e. those passed to `logger.Noticef`) should begin with a capital letter, and use "Cannot X" rather than "Error Xing":

Reviewer: ```suggestion
**Log messages** (that is, those passed to `logger.Noticef` or `logger.Debugf`) should begin with a capital letter, and use "Cannot X" rather than "Error Xing":
```
